packages/mdc-bottom-line/mdc-bottom-line.scss

```diff
@@ -14,29 +14,23 @@
 // limitations under the License.
 //
 
-@import "../functions";
+@import "./functions";
 @import "./mixins";
 @import "@material/theme/mixins";
 
-// postcss-bem-linter: define textfield
-.mdc-text-field {
-  &__bottom-line {
-    position: absolute;
-    bottom: 0;
-    left: 0;
-    width: 100%;
-    height: 2px;
-    transform: scaleX(0);
-    transition: mdc-text-field-transition(transform), mdc-text-field-transition(opacity);
-    opacity: 0;
-    z-index: 2;
-  }
+// postcss-bem-linter: define bottom-line
+.mdc-bottom-line {
+  position: absolute;
+  bottom: 0;
+  left: 0;
+  width: 100%;
+  height: 2px;
+  transform: scaleX(0);
+  transition: mdc-text-field-transition(transform), mdc-text-field-transition(opacity);
```

Comment:
we shouldnt have any references to text-field in this file. Why is `mdc-text-field-transition` here?